python/nav/mibs/entity_mib.py

```diff
@@ -199,6 +199,10 @@ def is_power_supply(entity):
     def is_fan(entity):
         return entity["entPhysicalClass"] == "fan"
 
+    def is_transceiver(self, entity):
+        transceiver_in_name = "transceiver" in entity['entPhysicalDescr'].lower()
+        return transceiver_in_name and bool(self.get_nearest_port_parent(entity))
```

Comment:
Coercing to `bool()` isn't necessary to evaluate the truthiness of the return value. IMHO it reads better without :)

```suggestion
        return transceiver_in_name and self.get_nearest_port_parent(entity)
```